website/build.gradle

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Define common lifecycle tasks and artifact types
+apply plugin: "base"
+
+task cleanWebsite(type: Delete) {
+  delete '.sass-cache', '.testcontent', 'content'
```

Comment:
I tried, this, and it didn't seem to work. Specifically, I ran: `./gradlew :beam-website:build; ./gradlew :beam-website:clean; ls website/`, and the above directories were still there.

I suspect this may be automatic for some Gradle plugins, but it seems not in the general case.